### ISSUE TITLE
Investigate — WP 6.6 Compatibility

### ISSUE THREAD

**Issue by jaymcp** [amnestywebsite/humanity-theme#365]
Exploratory ticket for discovering issues introduced by upgrading our supported WP version from 6.4.* to 6.6.*.

Issues found:
- superfluous padding on the frontend. Can be fixed by removing `styles.spacing.padding` from `theme.json`. Removing this, however, results in a lack of padding in the block editor.
- custom theme translations loaded by our translation plugin no longer load
- shop front has some issues (products not loading)
- php templates are no longer selectable in the block editor

**Comment by jaymcp** [comment]
> php templates are no longer selectable in the block editor

this appears to be an intentional change in WP 6.5, and there doesn't appear to be a solution. workaround is to use quick edit to select a template.
this is terrible ux for editorial, so it increases the priority of #246

**Comment by jaymcp** [comment]
> superfluous padding on the frontend. Can be fixed by removing styles.spacing.padding from theme.json. Removing this, however, results in a lack of padding in the block editor.

solution: keep padding in theme.json, and overwrite it on the frontend with css, as that's easier than overwriting editor styles.

**Comment by jaymcp** [comment]
> shop front has some issues (products not loading)

as per my above [comment](https://github.com/amnestywebsite/humanity-theme/issues/365#issuecomment-2304993061), we need to set the shop page template back via SQL, thanks to WP deciding to no longer support classic/hybrid theme php templates in 6.5

```sql
update wp_postmeta set meta_value = 'templates/shop-index.php' where meta_key = '_wp_page_template' and post_id = [id]
```

**Comment by jaymcp** [comment]
> Issues found:
> 
> * superfluous padding on the frontend. Can be fixed by removing `styles.spacing.padding` from `theme.json`. Removing this, however, results in a lack of padding in the block editor.

This is fixed in core in WP 6.6.2 https://core.trac.wordpress.org/ticket/61704
